WordPress/Classes/Models/SiteInfo.swift

```diff
@@ -1,9 +1,9 @@
 import Foundation
 
 @objc open class SiteInfo: NSObject {
-    var name = ""
-    var tagline = ""
-    var url = ""
-    var hasJetpack = false
-    var icon = ""
+    @objc var name = ""
+    @objc var tagline = ""
+    @objc var url = ""
+    @objc var hasJetpack = false
+    @objc var icon = ""
```

Comment:
😮 wow, Swift 4 is definitely a nudge to get rid of all Obj-C code, that's a lot of `@objc`

Reply:
It's painful, and i feel ashamed about it!